token/js/src/actions/createAccount.ts

```diff
@@ -26,10 +26,11 @@ export async function createAccount(
     owner: PublicKey,
     keypair?: Keypair,
     confirmOptions?: ConfirmOptions,
-    programId = TOKEN_PROGRAM_ID
+    programId = TOKEN_PROGRAM_ID,
+    idemPotent = false,
```

Comment:
We already have a lot of null / undefined parameters in this function, so to simplify usage, let's have a separate `createAssociatedTokenAccountIdempotent` action which only does a create idempotent call in `creteAssociatedTokenAccount.ts`, and not have the flag here.